StatBot/Models/DataModel.cs

```diff
@@ -20,6 +21,16 @@ public static List<User> Users
             set { _users = value; }
         }
 
+        public static List<Statistic> Statistics
+        {
+            get
+            {
+                var db = new PostgresStatsRepository();
+                return db.GetAll().ToList();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor 'Statistics' into a method, properties should not copy collections. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS2365)